docs/src/migrating.md

```diff
@@ -0,0 +1,27 @@
+Migration
+=========
+
+How to migrate from v0.10 to v0.11
+----------------------------------
+
+v0.11 has a few subtle breaking changes to `eof` and `seekend`.
+
+### `Memory(data::ByteData)`
+
+The `Memory(data::ByteData)` constructor was removed.
+Use `Memory(pointer(data), sizeof(data))` instead.
+
+### `seekend(stream::TranscodingStream)`
+
+Generic `seekend` for `TranscodingStream` was removed.
+Use `skip(stream, typemax(Int64))` instead.
```

Comment:
```suggestion
If the objective is to discard all remaining data in the stream, use `skip(stream, typemax(Int64))` instead where `typemax(Int64)` is meant to be a large number to exhaust the stream.
Ideally, specific implementations of `TranscodingStream` will implement `seekend` only if efficient means exist to avoid fully processing the stream.
```